### ISSUE TITLE
CLIPScore

### ISSUE THREAD

**Issue by MigCas31** [allenai/satlas-super-resolution#52]
Hi, 

Thank you for this interesting repository and contributions.
I clone the repo and tried to reproduce the results. However, I had higher values of CLIPScore than the paper claims. So, I undertake a little study to better understand the results of CLIPScore. 
I used the clipa-ViT-bigG-14 pretrained on the DataComp-1B on different pairs of images : 

![Image](https://github.com/user-attachments/assets/dcbd324e-836d-45d2-968e-6b4593b5f0c7)

Respectively : semantically different, same, ill-generated and mid- generated images (mid-generated means well generated but not perfectly)

In order to compute the ClipScore, I follow the provided pipeline (F.interpolate(), image_encode() and F.cosine_similarity).
For this pairs I have the following results : (approx. 10**-2)
- semantically different : 0,67
- same picture : 1.0
- ill-generated : 0,91
- mid-generated : 0,83
The remote sensing were generated with the pretrained weights for 8 inputs on the val_set with the configuration proposed (esrgan_baseline_8S2.yml), so the small esrgan proposed.

Thank you for your time, 
Hope you can help me

**Comment by piperwolters** [comment]
Hello! Thanks for your detailed issue, and apologies for the late response.

At a high-level, I also saw that CLIP scores were relatively "higher" than what you'd expect. For example, the photos of the laptop and the dog getting a score of 67 is not exactly intuitive. And with the remote sensing images, I found that most image pairs would get at least a score of 75, and the score would matter more the closer you got to 100. The score for images 5 and 6 is a bit surprising, as not even the patterns are similar. The score for images 7 and 8 seems reasonable, given the pattern is similar. 

At this point, more research on image similarity metrics has been released, and CLIP score should likely not be the only metric used to assess methods. It would be ideal to use metrics that looks at different aspects of the remote sensing imagery (landcover proportions, object accuracy and realism, correct colors, etc.).
